app/src/main/java/io/pslab/fragment/BaroMeterDataFragment.java

```diff
@@ -564,4 +612,12 @@ private void initiateBaroSensor(int type) {
                 break;
         }
     }
+
+    private float getAltitude(float pressure) {
+        if (pressure <= 0.0) {
+            return  0;
+        } else {
+            return (float) (44330 * (1 - Math.pow(pressure, 1.0 / 5.255)));
```

Comment:
Where did you get these values and equations? Better mention reference as a comment

Reply:
https://cdn-shop.adafruit.com/datasheets/BST-BMP180-DS000-09.pdf check section 3.6 on page 16